docs/intro.rst

```diff
@@ -0,0 +1,4 @@
+Introduction
```

Comment:
@perrygeo also a `.. todo::`?  I didn't see this linked from any of the other pages here; perhaps I missed something?